en/FileDeploys/index.md

```diff
@@ -0,0 +1,148 @@
+---
+lastUpdated: 2018-10-24
+---
+
+# File Deploys
+
+The **File Deploys** deploys files to enebular agent.  
+The use cases, for example, running shell script in enebular agent, adding setting files and so on.
```

Comment:
A use case example can be running shell script in enebular agent or adding setting files.